packages/lexical-table/src/LexicalTableSelectionHelpers.ts

```diff
@@ -371,6 +378,40 @@ export function applyTableHandlers(
     ),
   );
 
+  tableObserver.listenersToRemove.add(
+    editor.registerCommand<KeyboardEvent | ClipboardEvent | null>(
+      CUT_COMMAND,
+      (event) => {
+        const selection = $getSelection();
+        if (selection) {
+          if ($isNodeSelection(selection)) {
+            return false;
+          }
+
+          copyToClipboard(
+            editor,
+            objectKlassEquals(event, ClipboardEvent)
+              ? (event as ClipboardEvent)
+              : null,
+          );
+
+          if ($isTableSelection(selection)) {
+            $deleteCellHandler(event);
+            return true;
+          } else if ($isRangeSelection(selection)) {
+            editor.update(() => {
+              $deleteCellHandler(event);
+              selection.removeText();
+            });
```

Comment:
nit: editor.update is redundant in a command listener

```suggestion
            $deleteCellHandler(event);
            selection.removeText();
```

Reply:
updated